File: c/evenodd.c
//Odd or even program in C 
#include <stdio.h>
 
int main()
{
   int n;
 
   printf("Enter an integer\n");
   scanf("%d", &n);
 
   if (n%2 == 0)
      printf("Even\n");
   else
      printf("Odd\n");
 
   return 0;
}

File: java/EvenOdd.java
//This java program finds if a number is odd or even

import java.util.Scanner;

class EvenOdd {
    public static void main(String[] args) {
        int c;
        System.out.println("Input an integer");
        Scanner in = new Scanner(System.in);
        c = in.nextInt();

        if ((c / 2) * 2 == c)
            System.out.println("Even");
        else
            System.out.println("Odd");
    }
}
